blazar/plugins/oshosts/host_plugin.py

```diff
@@ -897,14 +932,16 @@ def notification_callback(self, event_type, payload):
                     ['reservable == 0',
                      'hypervisor_hostname == ' + data['host']])
                 if recovered_hosts:
-                    db_api.host_update(recovered_hosts[0]['id'],
-                                       {'reservable': True})
+                    self.set_reservable(recovered_hosts[0], True)
                     LOG.warn('%s recovered.',
                              recovered_hosts[0]['hypervisor_hostname'])
 
         return reservation_flags
 
     def set_reservable(self, resource, is_reservable):
+        if resource.get('disabled', False):
+            LOG.warn(f"{resource['hypervisor_hostname']} is disabled - cannot set reservable")
```

Comment:
I would set this to `LOG.debug`, as it will print every time the monitor runs in most cases.